docs/deployment/manual.md

```diff
@@ -88,3 +79,42 @@ $ kubectl get no -o json | jq .items[].metadata.labels
   "kubernetes.io/os": "linux",
 ...
 ```
+
+# Uninstallation
+
+If you followed the deployment instructions from the [Quick start](index) you
+can simply do:
+
+```bash
+kubectl -n nfd-operator delete NodeFeatureDiscovery my-nfd-deployment
+```
+
+Optionally, you can also remove the namespace:
+
+```bash
+kubectl delete ns nfd-operator
+```
+
+See the [node-feature-discovery-operator][nfd-operator] and [OLM][OLM] project
+documentation for instructions for uninstalling the operator and operator
+lifecycle manager, respectively.
+
+## Removing feature labels
```

Comment:
I think this could be a separate main section (i.e. separate file). It kinda applies to Helm deployment as well. WDYT?

Reply:
Like an uninstall section?